content/maxwell1_fundamentals/interface_conditions/index.rst

```diff
@@ -0,0 +1,48 @@
+
++---------------------------------+------------------------------------------------------------+---------------------------------------------------------------+
+|Property                         | Normal Component                                           | Tangential Component                                          |
++=================================+============================================================+===============================================================+
+|:math:`\mathbf{e}`               |:math:`\varepsilon_1e_1^\perp-\varepsilon_2e_2^\perp=\tau_f`|:math:`e_1^\parallel-e_2^\parallel=0`                          |
++---------------------------------+------------------------------------------------------------+---------------------------------------------------------------+
+|:math:`\mathbf{d}`               | :math:`d_1^\perp-d_2^\perp=\tau_f`                         |:math:`\varepsilon_2d_1^\parallel-\varepsilon_1d_2^\parallel=0`|
++---------------------------------+------------------------------------------------------------+---------------------------------------------------------------+
+|:math:`\mathbf{j}` (steady-state)| :math:`j_1^\perp=j_2^\perp`                                |:math:`\rho_1j_1^\parallel-\rho_2 j_2^\parallel=0`             |
++---------------------------------+------------------------------------------------------------+---------------------------------------------------------------+
+|:math:`\mathbf{h}`               |:math:`\mu_1h_1^\perp-\mu_2h_2^\perp=0`                     |:math:`h_1^\parallel-h_2^\parallel=K_f`                        |
++---------------------------------+------------------------------------------------------------+---------------------------------------------------------------+
+|:math:`\mathbf{b}`               | :math:`b_1^\perp-b_2^\perp=0`                              |:math:`\mu_1^{-1} b_1^\perp- \mu_2^{-1}b_2^\perp=K_f`          |
++---------------------------------+------------------------------------------------------------+---------------------------------------------------------------+
+
+	- **Normal Component** (:math:`\perp`): Components of fields or fluxes perpendicular to the interface.
+	- **Tangential Component** (:math:`\parallel`): Components of fields or fluxes parallel to the interface.
+	- **Free Surface Charge Density** (:math:`\tau_f`): The free surface charge per unit area on the interface. Occurs due to discontinuities in the normal component of the electric displacement across the interface.
+	- **Free Surface Current Density** (:math:`K_f`): The free surface current per unit area on the interface. Occurs due to discontinuities in the tangential component of the magnetic field across the interface.
+
+
+Derivation of Interface Conditions
```

Comment:
Including this title here creates an extra layer of depth to the table of contents that I think is likely confusing. 
![image](https://cloud.githubusercontent.com/assets/6361812/24765236/dabf8b12-1aab-11e7-93d2-d4cb235f69e5.png)

I would suggest replacing 
```

Derivation of Interface Conditions
----------------------------------
```
with 
```
**Contents**
```
as that is what we have used elsewhere for listing tables of contents. Also, using Bold does not introduce a new layer in the table of contents